frontend/src/stores/OutputStore.js

```diff
@@ -195,6 +190,87 @@ class OutputStore {
             return dataset.metadata.name;
         }
     }
+
+    @computed get dichotomousdeviance() {
```

Comment:
i dont normally put this stuff in the store b/c it's only used in one place so i put on the component. 

I'd recommend moving this to components instead. Currently the logic is split so I have to change code in multiple places to make a modification; your building an array in the store, unpacking it in a function inside the compoent, and then applying formatting. it'd be better just to do it all in the component and not build temporary array structures i think

Reply:
moved all the store functions to components.